examples/vite/yarn.lock

```diff
@@ -14,40 +14,6 @@
   resolved "https://registry.yarnpkg.com/@braintree/sanitize-url/-/sanitize-url-6.0.4.tgz#923ca57e173c6b232bbbb07347b1be982f03e783"
   integrity sha512-s3jaWicZd0pkP0jf5ysyHUI/RE7MHos6qlToFcGWXVp+ykHOy77OUMrfbgJ9it2C5bow7OIQwYYaHjk9XlBQ2A==
 
-"@cspotcode/source-map-support@^0.8.0":
```

Comment:
Why some many lines are removed from the lock file?

Reply:
That's interesting, must've committed something by accident. I'll revert this back to what it was.